pilot/pkg/kube/inject/inject.go

```diff
@@ -212,6 +212,7 @@ type Params struct {
 	Privileged                   bool                   `json:"privileged"`
 	SDSEnabled                   bool                   `json:"sdsEnabled"`
 	EnableSdsTokenMount          bool                   `json:"enableSdsTokenMount"`
+	PodDNSSearchNamespaces       []string               `json:"podDNSSearchNamespaces"`
```

Comment:
I am not sure why this is needed? is it just for testing?

Reply:
Yes. Just for testing